### DISCUSSION TITLE
Clear Filter URLs

### DISCUSSION THREAD

**Discussion by brandonzylstra** [avo-hq/avo#2430] [Feature requests]
I am looking for a way to link to a filtered Avo UI showing just the employees who report to a specific supervisor.  I would expect to use something like this:

https://example.com/tables/resources/employees?page=1&filter-employees-of=USER1234567

but instead the filters produce something like this:

https://example.com/tables/resources/employees?page=1&filters=eyJEaXJlY3RSZXBvcnRzRmlsdGVyIjoiNzQ5NTU4In0%3D

Is there any way to link to Avo and have the view filtered how my users need it?

**Comment by adrianthedev** [comment]
The first thing that comes to mind is to create a different route that redirects to what you need.

You can use [the helpers](https://docs.avohq.io/3.0/basic-filters.html#manually-create-encoded-urls) to encode/decode filter params.


I can't remember exactly why we chode to encode those params, but maybe it was a bad decision.

**Reply by brandonzylstra** [reply]
Thanks, I'll try the helpers!

Maybe the decision was based on URL limits in browsers like IE (and I think maybe early versions of Edge) that were around 2048.  If someone had lots of complex filter data, I suppose it might be possible to hit this limit.  

Another possibility might be the risk of query string parameter collisions if someone is already using query strings for lots of things.  (But this only seems like a possibility in a future where Avo components are mixed into the rest of the UI.)
